Assets/Scripts/VictoryDetector.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VictoryDetector
+{
+    public GameState CheckVictoryPlayers(List<FieldValue> fieldValues, TextOutput textOutput, FieldValue fieldValue,
+        GameState gameState)
```

Comment:
why do you pass gameState here? it is an implicit and unnecessary logic, you could determine who won by fieldValue passed. And if this logic repeats anywhere else you can just move it into an extension